src/main/drivers/flash_m25p16.c

```diff
@@ -70,7 +72,11 @@ static bool isLargeFlash = false;
 #define SECTOR_ERASE_TIMEOUT_MILLIS  5000
 #define BULK_ERASE_TIMEOUT_MILLIS    21000
 
-static flashGeometry_t geometry = {.pageSize = M25P16_PAGESIZE};
+#define M25P16_PAGESIZE 256
+
+static flashGeometry_t geometry = {.pageSize = M25P16_PAGESIZE, .flashType = FLASH_TYPE_NOR };
```

Comment:
These get a bit easier to read when the elements are formatted one per line. Also, convention (or at least most common pattern in Betaflight) is to have a space inside the curly braces when using the one line format.